### ISSUE TITLE
Cannot set UMS StructureTag and DeviceAttributes via Update-UMSDevice

### ISSUE THREAD

**Issue by rlke42** [falkheiland/PSIGEL#41]
Theory would be to add UMSStructureTag (with String value) and DeviceAttributes (with Hashtable/PSObject value) as new Parameters of Update-UMSDevice.

Example: 

```
$NewUMSStructureTag = "NewValue"
$ChangedDeviceAttributes = @{
  "DeviceAttribute1Name" = "DeviceAttribute1Value";
  "DeviceAttribute2Name" = "DeviceAttribute2Value"
}

$TestDevices = @("T101L001"; "T101T001")
$WebSession = New-UMSAPICookie -Computername $UMSServerFQDN -Credential (Get-Credential)
$Devices = Get-UMSDevice -Computername $UMSServerFQDN -TCPPort $UMSServerPort -WebSession $WebSession -Filter details | Where-Object { $_.Name -cin $TestDevices }
foreach ($Device in $Devices) {
  Update-UMSDevice -Computername $UMSServerFQDN -TCPPort $UMSServerPort -WebSession $WebSession -Id $Device.Id -UMSStructureTag $NewUMSStructureTag -DeviceAttributes $ChangedDeviceAttributes
}
```

**Comment by rlke42** [comment]
UMS StructureTag cannot be integrated as the API currently does not allow this value to be edited via IMI (see https://kb.igel.com/igelimi-v3/en/put-v3-thinclients-tcid-2723464.html)

**Comment by rlke42** [comment]
Missing DeviceAttributes fixed by #42
UMS Structuretag is not fixable as it cannot be set via IMI at the Moment
